src/EditorFeatures/Test/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -111,5 +114,14 @@ internal override IEnumerable<Diagnostic> GetDiagnostics(TestWorkspace workspace
                 }
             }
         }
+
+        private void AssertNoAnalyzerExceptionDiagnostics(IEnumerable<Diagnostic> diagnostics)
+        {
+            // The internal method `Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor::IsAnalyzerExceptionDiagnostic()`
+            // does essentially this, but due to linked files between projects, this project cannot have internals visible
```

Comment:
I think we've had a few extern aliases added to deal with messes like this. Or how hard is the underlying problem to fix?